serving/central/src/main/webapp/components/UploadFab.jsx

```diff
@@ -47,7 +47,7 @@ export default function UploadFab() {
        if (selectedFile){
            alert("Your file is being uploaded!")
 
-           axios.post("http://"+window.location.host+"/uploading/models?modelName="+selectedFile.name)
+           axios.get("http://"+window.location.host+"/uploading/models?modelName="+selectedFile.name)
```

Comment:
Post is right. The convention with HTTP methods is to use post to create things (https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods). You won't be able to use the query parameters though (no ?modelName=...). See the post example on https://github.com/axios/axios

Reply:
This change was meant to be apart of another commit. But, you are correct.